bccsp/schemes/dlog/crypto/signature.go

```diff
@@ -812,13 +1026,19 @@ func (sig *Signature) Ver(
 	// one bigint (hash of the issuer public key) of length math.FieldBytes
 	// disclosed attributes
 	// message that was signed
-	pdl := len([]byte(signLabel)) + 7*(2*curve.FieldBytes+1) + curve.FieldBytes + len(Disclosure) + len(msg) + ProofBytes[RevocationAlgorithm(sig.NonRevocationProof.RevocationAlg)]
-	if verifyEIDNym {
-		pdl = len([]byte(signWithEidNymLabel)) + 9*(2*curve.FieldBytes+1) + curve.FieldBytes + len(Disclosure) + len(msg) + ProofBytes[RevocationAlgorithm(sig.NonRevocationProof.RevocationAlg)]
+	pdl := curve.FieldBytes + len(Disclosure) + len(msg) + ProofBytes[RevocationAlgorithm(sig.NonRevocationProof.RevocationAlg)]
```

Comment:
Same as comment above, it would be great to document these lengths for posterity

Reply:
Sounds good, do you have a suggestion of what clarification I should add to the existing comments?

Reply:
see here

https://github.com/IBM/idemix/pull/25#discussion_r1116642712